app/scripts/museum/museum-service.js

```diff
@@ -0,0 +1,66 @@
+
+(function () {
+    'use strict';
+
+    /* ngInject */
+    function Museum ($http, Config) {
+        var sqlApi = ['https://', Config.cartodb.account,
+                      '.cartodb.com/api/v2/sql'].join('');
+        var cols = {
```

Comment:
It may be helpful to expose this as a public `angular.constant()` value such as `DBColumns` so it can be used elsewhere.